public/admin/config.yml

```diff
@@ -154,3 +154,66 @@ collections:
                 fields:
                   - { label: Color, name: color, widget: string }
                   - { label: Content, name: content, widget: markdown }
+  - name: learn
+    label: Learn
+    format: json
+    description: Learn content
+    slug: '{{slug}}'
+    editor:
+      preview: true
+    files:
+      - name: Learn-Landing
+        label: 'Learn: Landing page'
```

Comment:
Consider renaming to `Landing page` (without the "Learn:") prefix so it doesn't look so repetitive in the CMS